tests/framework/defs.py

```diff
@@ -3,7 +3,7 @@
 """Some common defines used in different modules of the testing framework."""
 API_USOCKET_URL_PREFIX = 'http+unix://'
 """URL prefix used for the API calls through a UNIX domain socket."""
-API_USOCKET_NAME = 'api.socket'
+API_USOCKET_NAME = 'tmp/firecracker.socket'
```

Comment:
Try:
```suggestion
API_USOCKET_NAME = '/tmp/firecracker.socket'
```
But this is a poor fix as it assumes that firecracker will create the api sock at that default location (inside the jail).

A nicer solution would be to leave `API_USOCKET_NAME = 'api.socket'` like it was, and add code in the test framework to call the jailer with the right parameters as to forward this api socket path to the firecracker process. That way the test framework is self-contained and independent of firecracker default values.

Reply:
nice suggestion @acatangiu , will make test framework to call the jailer with the right parameters 👍

Reply:
@acatangiu changed code as like you suggested 🙂